packages/ui-utils/src/types.ts

```diff
@@ -1,3 +1,4 @@
+import { MIMEType } from 'node:util';
```

Comment:
We cannot have node imports. This is potentially used on the client as well

Reply:
deleted unused import